universal-application-tool-0.0.1/app/auth/AuthIdentityProviderName.java

```diff
@@ -0,0 +1,31 @@
+package auth;
+
+import java.util.Optional;
+
+/** Enum representing supported options for auth IDP for applicants and admins. */
+public enum AuthIdentityProviderName {
+  IDCS_APPLICANT("idcs"),
+  LOGIN_RADIUS_APPLICANT("login-radius"),
+  ADFS_ADMIN("adfs-admin"),
+  ;
```

Comment:
missing } ?

Reply:
er sorry I missed the structure here and the   , } tripped me up

Reply:
surprisingly, no. enums are weird. there was an extra comma that I removed though.